Sources/Components/Components/Watchlist/WKWatchlistViewModel.swift

```diff
@@ -13,15 +13,27 @@ public final class WKWatchlistViewModel {
 			self.filter = filter
 		}
 	}
+    
+    public struct Configuration {
```

Comment:
Just thinking out loud – certainly fine as-is, but trying to think if we anticipate having more stuff in this struct or if something like `PresentationConfiguration` that explicitly provides context about the content in here would make more sense.

Reply:
Nah I didn't anticipate anything more. I pushed this change - let me know if this is what you were thinking. If config additions are needed outside of presentation, we can always change the naming back to what it was.